.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 # run for only for pull requests or master branch
 if: 'type = pull_request OR branch = master'
```

Comment:
Not sure if exclusion could've been added to this line as well 🤔

Reply:
Like this? 

`if: '(type = pull_request AND branch != weblate:weblate-trustroots-translation) OR branch = master'`

Not sure either if it will work. But for me, any option looks good